_episodes/17-performance.md

```diff
@@ -10,6 +10,11 @@ objectives:
 keypoints:
 - "(Reflective) Practice makes perfect."
 ---
+
+Continuing our theme of developing practical Carpentries teaching skills,
+this section provides another chance to practice live coding, observe others
+and give/receive feedback.
+
```

Comment:
maybe add something about incorporating or responding to feedback?

Reply:
ah, yes!